src/extensions/renderer/canvas/drawing-images.js

```diff
@@ -23,9 +23,11 @@ CRp.drawInscribedImage = function( context, img, node, index ){
   var nodeY = pos.y;
   var fit = getIndexedStyle( node, 'background-fit', 'value', index );
   var repeat = getIndexedStyle( node, 'background-repeat', 'value', index );
-  var paddingX2 = node.pstyle( 'padding' ).pfValue * 2;
-  var nodeW = node.width() + ( getIndexedStyle( node, 'background-width-relative-to', 'value', index ) === 'inner' ? 0 : paddingX2 );
-  var nodeH = node.height() + ( getIndexedStyle( node, 'background-height-relative-to', 'value', index ) === 'inner' ? 0 : paddingX2 );
+  var nodeW = node.width();
+  var nodeH = node.height();
+  var paddingX2 = util.computePaddingValues(nodeW, nodeH, node.pstyle( 'padding' ), node.pstyle( 'padding-relative-to' ).value ) * 2;
+  var nodeTW = nodeW + ( getIndexedStyle( node, 'background-width-relative-to', 'value', index ) === 'inner' ? 0 : paddingX2 );
```

Comment:
Ok I see so percentage is only for use on parent nodes so autopadding is fine. Will update commit thanks!